releasenotes/notes/support_excitedstatessolver_bopes-2bbb5a282b1ff582.yaml

```diff
@@ -0,0 +1,18 @@
+features:
+  - |
+    The solver of BOPESSampler now accepts solvers of type
+    `Union[GroundStateSolver,ExcitedStatesSolver]` instead of only a `GroundStateSolver`. 
+    This generalizes to excited states the sampling 
+    of nuclear coordinates for Born Oppenheimer Potential Energy Surfaces.
+    Adds the `get_qubit_operators()` in
+    :class:`qiskit_nature.algorithms.ExcitedStatesEigensolver`
+    This matches the format of
+    :class:`qiskit_nature.algorithms.GroundStatesEigensolver`
+    BOPESSampler now also accepts auxiliary operators to pass to the solver.
+    Geometry-dependent observable can also be defined as auxiliaries
+
+upgrade:
+  - |
+    The tutorial 05 now includes calculations of excited state Born Oppenheimer potential energy
```

Comment:
The tutorial numbers are not really visible in the online documentation. Instead, you can link to the tutorial directly, as done e.g. here:
https://github.com/Qiskit/qiskit-nature/blob/e5b5021863a4f7d8658d4f72959217347b078053/releasenotes/notes/bogoliubov-transform-885eb6442172c76e.yaml#L8

Reply:
Done